modules/apps_api/lib/apps_api/notification_service.rb

```diff
@@ -46,35 +48,53 @@ def parse_event(event)
         app_record = DirectoryApplication.find_by(name: event['actor']['displayName'])
       end
       user = @okta_service.user(user_id)
-      create_hash(app_record: app_record, user: user, published: event['published'])
+      create_hash(app_record: app_record, user: user, event: event)
     end
 
-    def create_hash(app_record:, user:, published:)
+    def create_hash(app_record:, user:, event:)
       {
+        'uuid' => event['uuid'],
         'app_record' => app_record,
         'user_email' => user.body['profile']['email'],
         'options' => {
           'first_name' => user.body['profile']['firstName'],
           'application' => app_record ? app_record['name'] : nil,
-          'time' => published,
+          'time' => format_published_time(event['published']),
           'privacy_policy' => app_record ? app_record['privacy_url'] : nil,
           'password_reset' => Settings.vanotify.links.password_reset,
           'connected_applications_link' => Settings.vanotify.links.connected_applications
         }
       }
     end
 
+    def format_published_time(published)
+      # Formats iso8601 time stamp into readable language
+      # 2020-11-29T00:23:39.508Z -> 11/29/2020 at 00:23:39:23AM
+      Time.zone.parse(published).strftime('%m/%d/%Y at %T:%M%p')
+    end
+
     def event_is_invalid(event)
```

Comment:
nit: event_is_invalid?
since this is asking a question and the return is always either true or false

secondary nit: this might be just personal preference, but i almost always think the positive = true concept works better. so, "event_is_valid?" instead of "event_is_invalid?"

Reply:
sort of the same thing with several of these methods below as well. ?'s at the end and positive "event_successful?" instead of "event_unsuccessful?"

Reply:
i tend to always write these helper method as a way to filter out failures, rather than to confirm successes, if that makes sense. I think it has to do with preferring the `unless foo_is_bad, continue` method over the `if foo_is_good, continue` way, if that makes sense. I'm with you on changing the method name to include ? though all the way

Reply:
fair enough, the positive `if foo_is_good, continue` makes a ton more sense to my brain but definitely a nitpick.